### ISSUE TITLE
is there a way to manually select coin

### ISSUE THREAD

**Issue by truckinCowboy** [RainbowMiner/RainbowMiner#2826]
Is there a way to manually select the coin and pool to mine or is it always going to pick it for me?
I have looked for this answer and can't seem to find what im looking for.

**Comment by whitindllc** [comment]
what pool or coin.... the answer is probably yes....

**Comment by truckinCowboy** [comment]
any pool or coin i am having problems forcing only 1 coin at a time it is doing what it wants instead
i disabled the autocoin and input the4 coin that i want and it still does what it wants

**Comment by truckinCowboy** [comment]
need a input and button on the api locking a coin in would be great

**Comment by whitindllc** [comment]
give me an example

for example if you have zpool, zergpool, rplant and WTF ever pool setup and you just want to mine meow at rplant even though the other pools also have that coin... if you have all the wallets setup properly you go to pools.config.txt and you find the pool that you want to force mining to and you go to the section where focuswallet is.... and in between the " " you put in the coin you want to focus on....

if its a bunch of coins in the same algo you up to algorithm and pick the one you want....

now if you just want to mine a certain algo and want RBM to chose the best one you go into the gui and then setup on the left.... pools tab in the middle.. then chose the algorithm... you can also choose which ones to exclude as well

![image](https://github.com/RainbowMiner/RainbowMiner/assets/128111946/1efdb8b7-58f8-41ee-8704-dbbd5bc05bb8)

**Comment by truckinCowboy** [comment]
yeah i get all that i just didnt want to jump thru 10 hoops everytime i want to force it on a coin.
I guess i am suggesting a feature
